# Essa informação é utilizada para
# represetar um corpo rígido (objeto: robô ou bola) em um espaço do Futebol de Robôs, sendo necessário apenas:
# posição X e Y e rotação em Z.

float32 x
float32 y
float32 yaw
